index.bs

```diff
@@ -1302,6 +1302,12 @@ shared among all [=environment settings objects=].
 Note: This would ideally use <a spec=storage>storage bottles</a> to provide access to the attribution caches.
 However attribution data is inherently cross-site, and operations on storage would need to span across all storage bottle maps.
 
+An <dfn>internal ID</dfn> is an integer.
+
+To <dfn>get the next internal ID</dfn>, return an [=internal ID=] strictly
+greater than any previously returned by this algorithm. The user agent MAY reset
```

Comment:
Now that this is merged, are there any places in the spec that rely on this ordering?